pkg/kubelet/cm/helpers_linux.go

```diff
@@ -44,3 +47,46 @@ func GetCgroupSubsystems() (*CgroupSubsystems, error) {
 		MountPoints: mountPoints,
 	}, nil
 }
+
+// GetPodResourceRequest returns the pod requests for the supported resources.
+// Pod request is the summation of resource requests of all containers in the pod.
+func GetPodResourceRequests(pod *api.Pod) api.ResourceList {
+	requests := api.ResourceList{}
+	for _, container := range pod.Spec.Containers {
+		requests = quota.Add(requests, container.Resources.Requests)
```

Comment:
@derekwaynecarr 
It is odd that helper methods are in `quota` package. Should we move them to `pkg/api`, now that it is being used outside of quota pkg?